### PR TITLE
Update 2015-10-03-docker-install-mac.md

### PR DESCRIPTION
**Pull request by SyntaxStacks**
Added link to docker for mac.

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-42550c11ef/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-bf1e1ffd01/)

File: _tutorials/2015-10-03-docker-install-mac.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ topics:
 
 This tutorial describes how to install and set up Docker on Mac OS X.
 
+EDIT: [Docker for Mac](https://www.docker.com/products/docker#/mac) is availble for Apple macOS Yosemite 10.10.3 and above. Follow the instructions below to install docker on previous versions of Mac OS X.
```
Comment (everett-toews) [inline _tutorials/2015-10-03-docker-install-mac.md]: Change `EDIT` to `**Note**` please.

Comment (SyntaxStacks) [inline _tutorials/2015-10-03-docker-install-mac.md]: I have made the requested changes.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-e892540991/)

**Pull request merged**